setup.py

```diff
@@ -19,7 +19,7 @@
       include_package_data=True,
       zip_safe=False,
       install_requires=[
-        'django>=1.3.1',
```

Comment:
now that I think about it and based from other PRs, this library mat want to stay backwards compatible